### ISSUE TITLE
xcodeproj fails to initialize object from unknown ISA: PBXFileSystemSynchronizedRootGroup

### ISSUE THREAD

**Issue by xinsight** [CocoaPods/Xcodeproj#966]
Xcode 16, I added a folder "Fonts" to my project and `xcodeproj` now fails with the following:

> /opt/homebrew/lib/ruby/gems/3.0.0/gems/xcodeproj-1.25.0/lib/xcodeproj/project/object.rb:359:in `rescue in object_with_uuid': `PBXGroup` attempted to initialize an object with unknown ISA `PBXFileSystemSynchronizedRootGroup` from attributes: `{"isa"=>"PBXFileSystemSynchronizedRootGroup", "explicitFileTypes"=>{}, "explicitFolders"=>[], "path"=>"Fonts", "sourceTree"=>"<group>"}` (RuntimeError)

> If this ISA was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new

**Comment by xinsight** [comment]
Workaround: Use "New Group" instead of "New Folder".

**Comment by Flexible-Universe** [comment]
When I use 'New Group,' it works. Every newly created project in Xcode 16 now always creates folders. Is there already a timeline for when it will also work with folders?

**Comment by xinsight** [comment]
Looks like all these issues are the same Xcode 16 issue:
https://github.com/CocoaPods/Xcodeproj/issues/968
https://github.com/CocoaPods/Xcodeproj/issues/967
https://github.com/CocoaPods/Xcodeproj/issues/965
https://github.com/CocoaPods/Xcodeproj/issues/963

**Comment by AIFlowML** [comment]
Having same issues and more 

[Xcodeproj] Xcode project version (77) is higher than the latest supported by xcodeproj (63)

**Comment by xinsight** [comment]
Correct, when i added a folder, the xcodeproj version gets bumped from 54 -> 70. And when i remove the folder and use a group, it gets automatically downgraded.

```
--- a/XXX.xcodeproj/project.pbxproj
+++ b/XXX.xcodeproj/project.pbxproj
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 70;
+	objectVersion = 54;
 	objects = {
 
 /* Begin PBXAggregateTarget section */
 ```

**Comment by Varun-Sethi-Dev** [comment]
Same issue any fixes around it

**Comment by sapnabansaltrinet** [comment]
I am also facing the same issue with creation of new project in Xcode16

**Comment by thederrickbrown** [comment]
I am also facing the same issue with creation of new project in Xcode16, details:

### Command

```
/Users/username/.rbenv/versions/3.3.5/bin/pod init
```

### Report

* What did you do?

* What did you expect to happen?

* What happened instead?


### Stack

```
   CocoaPods : 1.15.2
        Ruby : ruby 3.3.5 (2024-09-03 revision ef084cc8f4) [x86_64-darwin21]
    RubyGems : 3.5.20
        Host : macOS 14.6.1 (23G93)
       Xcode : 16.0 (16A242d)
         Git : git version 2.46.2
Ruby lib dir : /Users/username/.rbenv/versions/3.3.5/lib
Repositories : trunk - CDN - https://cdn.cocoapods.org/
```

### Plugins

```
cocoapods-deintegrate : 1.0.5
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.1
cocoapods-trunk       : 1.6.0
cocoapods-try         : 1.2.0
```

### Error

```
RuntimeError - `PBXGroup` attempted to initialize an object with unknown ISA `PBXFileSystemSynchronizedRootGroup` from attributes: `{"isa"=>"PBXFileSystemSynchronizedRootGroup", "path"=>"myapp", "sourceTree"=>"<group>"}`
If this ISA was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:359:in `rescue in object_with_uuid'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:349:in `object_with_uuid'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:300:in `block (2 levels) in configure_with_plist'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:299:in `each'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:299:in `block in configure_with_plist'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:296:in `each'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:296:in `configure_with_plist'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project.rb:272:in `new_from_plist'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:350:in `object_with_uuid'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:290:in `block in configure_with_plist'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:287:in `each'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:287:in `configure_with_plist'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project.rb:272:in `new_from_plist'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project.rb:213:in `initialize_from_file'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project.rb:113:in `open'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/command/init.rb:41:in `validate!'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/claide-1.1.0/lib/claide/command.rb:333:in `run'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/command.rb:52:in `run'
/Users/username/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/bin/pod:55:in `<top (required)>'
/Users/username/.rbenv/versions/3.3.5/bin/pod:25:in `load'
/Users/username/.rbenv/versions/3.3.5/bin/pod:25:in `<main>'
```

**Comment by ninacoder-info** [comment]
@sapnabansaltrinet i just convert the OneSignalNotificationServiceExtension to Group and it working fine, but screw OneSignal its always the one cause trouble, really tired everytime using it

**Comment by amorde** [comment]
This is fixed by https://github.com/CocoaPods/Xcodeproj/pull/985 and shipped in [1.26.0](https://github.com/CocoaPods/Xcodeproj/releases/tag/1.26.0)

**Comment by thisisadithk** [comment]
> @sapnabansaltrinet i just convert the OneSignalNotificationServiceExtension to Group and it working fine, but screw OneSignal its always the one cause trouble, really tired everytime using it

This worked for me. Went to Xcode, right clicked on the OneSignalNoificationSeviceExtension folder (which would be in blue), clicked on the "Convert to Group" option, the color changes to grey. Ran pod install in the iOS folder and the build error was no longer coming. Successfully installed pods.
